client-haskell/src/Icepeak/Client.hs

```diff
@@ -50,6 +54,25 @@ data Client = Client
   , clientToken :: Maybe ByteString
   }
 
+data Durability
+  = DurabilityEventual
```

Comment:
Wouldn't `Effect` instead of `Durability` map better the idea involved here?

In any case, I prefer `EventualDurability` instead of `DurabilityEventual`.

Reply:
I don't like these names either, I just couldn't come up with something better at that time.

Reply:
I used `EventualDurability` now.